### DISCUSSION TITLE
Constrained estimation Explanatory/regress

### DISCUSSION THREAD

**Discussion by fbrazdik** [IRIS-Solutions-Team/IRIS-Toolbox#402] [Q&A] [answered]
Hi,

I am wondering if it is possible to run constrained estimation for the explanatory object

something like this 
expy0 = Explanatory.fromString("difflog(x) = @(1) + @(2)*difflog(x{-1}) + @(3)*log(z)");

and do the regression 
where @(1)+@(2)+@(3) =1

Is there a way?

Thanks,
Fero

**Comment by fbrazdik** [comment]
This may work :-)
expy0 = Explanatory.fromString("difflog(x) = @(1) + @(2)*difflog(x{-1}) + (1-@(1)-@(2))*log(z)");
but what for like @(1)>0, etc?

**Comment by jaromir-benes** [comment, marked as answer]
Unfortunately, not available...
